test/credis_test.py

```diff
@@ -1,14 +1,18 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
 
 import os
-import redis
 import unittest
 
+import redis
+
 import ray
 
 
+def ParseClient(addr_port_str):
```

Comment:
Naming convention in Python is `parse_client`.

Reply:
Done.